docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1781,6 +1781,14 @@
                 </tr>
 
                 <tr>
+                    <td class="property" id="stores-rocksdb-telemetry">stores.<span class="store">store-name</span>.<br>rocksdb.telemetry.list</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        A list of RocksDB properties to expose as metrics (gauges).
```

Comment:
Maybe link to the properties:
<a href="https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L409"> RocksDB properties</a>. 

The line number will change but I couldn't find them listed elsewhere in RocksDB documentation.

Reply:
Yeah - good idea, cheers.